logging/cluster-logging-loki.adoc

```diff
@@ -37,6 +37,8 @@ ifdef::openshift-enterprise[]
 endif::[]
 ////
 
+include::modules/logging-loki-access.adoc[leveloffset=+1]
```

Comment:
This isn't showing up in the preview because it should be
```
include::modules/logging-loki-log-access.adoc[leveloffset=+1]
```

Reply:
Whoops, thanks!